nx_parallel/algorithms/centrality/tests/test_betweenness_centrality.py

```diff
@@ -23,13 +23,41 @@ def get_chunk(nodes):
 
         return chunks
 
+    return get_chunk
+
+
+def get_chunk_edges_random(nodes):
+    num_chunks = nxp.cpu_count()
+    chunks = [[] for _ in range(num_chunks)]
+    for i, node in enumerate(nodes):
+        chunks[i % num_chunks].append(node)
+    return chunks
+
+
+def test_betweenness_centrality_get_chunks():
     G = nx.fast_gnp_random_graph(100, 0.1, directed=False)
     H = nxp.ParallelGraph(G)
     ebc = nx.edge_betweenness_centrality(G)
-    par_bc_chunk = nxp.betweenness_centrality(H, get_chunks=get_chunk)  # smoke test
+    par_bc_chunk = nxp.betweenness_centrality(
+        H, get_chunks=get_chunk_function(ebc)
+    )  # smoke test
     par_bc = nxp.betweenness_centrality(H)
 
     for i in range(len(G.nodes)):
         assert math.isclose(par_bc[i], par_bc_chunk[i], abs_tol=1e-16)
     # get_chunk is faster than default(for big graphs)
     # G = nx.bipartite.random_graph(400, 700, 0.8, seed=5, directed=False)
+
+
+def test_edge_betweenness_centrality_get_chunks():
```

Comment:
no need to add this test.

In PR https://github.com/networkx/nx-parallel/pull/63 I've added a single test that collects all functions with `get_chunks` kwarg and runs smoke tests for all of them with a random chunking passed into the `get_chunks` arg. (see [test_get_chunks.py](https://github.com/networkx/nx-parallel/blob/06916f9c14cd1c338319a0e9d305a1b2d75fe93c/nx_parallel/tests/test_get_chunks.py)). 

And if PR https://github.com/networkx/nx-parallel/pull/63 gets merged before this PR then please ensure to put `edge_betweenness_centrality` in the `chk_dict_vals` list in the `test_get_chunks.py` file. Otherwise, I'll make sure it's in the PR https://github.com/networkx/nx-parallel/pull/63.

Reply:
> And if PR https://github.com/networkx/nx-parallel/pull/63 gets merged before this PR then please ensure to put edge_betweenness_centrality in the chk_dict_vals list in the test_get_chunks.py file. Otherwise, I'll make sure it's in the PR https://github.com/networkx/nx-parallel/pull/63.

PR https://github.com/networkx/nx-parallel/pull/63 is merged now. 

Also, please make sure that you run `sh _nx_parallel/script.sh` to add this algorithm to the `get_info` function.